src/components/Controls.vue

```diff
@@ -124,11 +123,14 @@ export default {
           <v-card-title class="pb-6">MultiNet Node Link Controls</v-card-title>
 
           <v-card-text>
-            <v-card-subtitle class="pb-0 pl-0">Marker Type</v-card-subtitle>
-            <v-radio-group v-model="nodeMarkerType">
-              <v-radio name="active" label="Circle" value="Circle" @click="renderNested = false; nodeMarkerType = 'Circle'"></v-radio>
-              <v-radio name="active" label="Rectangle" value="Rectangle"></v-radio>                
-            </v-radio-group>
+            <v-card-subtitle class="pb-0 pl-0" style="display: flex; align-items: center; justify-content: space-between">
+              Render Nested
```

Comment:
Let's not use the phrase "render nested", as it's not super informative. Something like:

```suggestion
              Display charts
```

might work better.